ethexe/blob-loader/src/lib.rs

```diff
@@ -319,37 +324,32 @@ impl<DB: Database> BlobLoaderService for BlobLoader<DB> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use ethexe_common::gear_core::ids::prelude::CodeIdExt;
 
     #[tokio::test]
-    async fn test_read_code_from_tx_hash() -> Result<()> {
+    async fn test_read_code_from_tx_hash() {
         let consensus_cfg = ConsensusLayerConfig {
             ethereum_rpc: "https://hoodi-reth-rpc.gear-tech.io".into(),
             ethereum_beacon_rpc: "https://hoodi-lighthouse-rpc.gear-tech.io".into(),
             beacon_block_time: Duration::from_secs(12),
+            attempts: NonZeroU8::new(3).unwrap(),
         };
+
         let blobs_reader = ConsensusLayerBlobReader {
             provider: ProviderBuilder::default()
                 .connect(&consensus_cfg.ethereum_rpc)
-                .await?,
+                .await
+                .unwrap(),
             http_client: Client::new(),
             config: consensus_cfg,
         };
-        let expected_code_id = "0x94892c2d1acaeb2d47e2ea79fe580a5b41c534f21333be1a86fe611ba4e0b7dc"
-            .parse()
-            .unwrap();
-        let CodeAndIdUnchecked { code, code_id } = blobs_reader
-            .read_code_from_tx_hash(
-                expected_code_id,
+
+        blobs_reader
+            .read_blob_from_tx_hash(
                 "0xee7f0082b6ad2fb1d409f39e5b169e102c27e4cf86b69a8a4006224cc91b4ae3"
                     .parse()
                     .unwrap(),
-                Some(3),
             )
-            .await?;
-        assert_eq!(code_id, expected_code_id);
```

Comment:
point of this code was to take the blob data, calculate the hash (code_id) and compare it with the code_id that was specified in the transaction